### ISSUE TITLE
Compilation errors when building pytsk with tag sleuthkit-4.11.0

### ISSUE THREAD

**Issue by joachimmetz** [sleuthkit/sleuthkit#2508]
@bcarrier seeing you tagged https://github.com/sleuthkit/sleuthkit/releases/tag/sleuthkit-4.11.0, I tested building pytsk with it. Some findings that likely can be fixed before final release (uploading source distribution packages):

```
sleuthkit/tsk/fs/fs_open.c: In function ‘tsk_fs_open_img_decrypt’:
sleuthkit/tsk/fs/fs_open.c:209:21: error: format not a string literal and no format arguments [-Werror=format-security]
  209 |                     tsk_error_set_errstr(imageType);
      |                     ^~~~~~~~~~~~~~~~~~~~
sleuthkit/tsk/fs/fs_open.c:220:52: error: format not a string literal and no format arguments [-Werror=format-security]
  220 |                         tsk_error_set_errstr(result->desc);
      |                                              ~~~~~~^~~~~~
sleuthkit/tsk/fs/fs_open.c:224:52: error: format not a string literal and no format arguments [-Werror=format-security]
  224 |                         tsk_error_set_errstr(result->desc);
      |                                              ~~~~~~^~~~~~
```

```
sleuthkit/tsk/vs/mm_open.c:187:48: error: format not a string literal and no format arguments [-Werror=format-security]
  187 |                     tsk_error_set_errstr(result->desc);
      |                                          ~~~~~~^~~~~~
```

These calls seem to be missing a format string as defined https://github.com/sleuthkit/sleuthkit/blob/0239c5934e348699d0be38f694fb6320252a91fc/tsk/base/tsk_base.h#L90

**Comment by joachimmetz** [comment]
I also would strongly encourage to address https://github.com/sleuthkit/sleuthkit/pull/2507 before a new release

**Comment by bcarrier** [comment]
I think we're good with this and the three you mentioned.

**Comment by joachimmetz** [comment]
thx much appreciated

**Comment by joachimmetz** [comment]
this issue has bee resolved in the release-4.11.0 branch/tag
